stdlib/EvaluatorSettings.pkl

```diff
@@ -91,6 +91,12 @@ rootDir: String?
 /// Configuration of outgoing HTTP requests.
 http: Http?
 
+/// Configuration for external module reader processes.
+externalModuleReaders: Mapping<String, ExternalReader>?
+
+/// Configuration for external resource reader processes.
+externalResourceReaders: Mapping<String, ExternalReader>?
```

Comment:
```suggestion
/// Configuration for external module reader processes.
@Since { version = "0.27.0" }
externalModuleReaders: Mapping<String, ExternalReader>?

/// Configuration for external resource reader processes.
@Since { version = "0.27.0" }
externalResourceReaders: Mapping<String, ExternalReader>?
```